### ISSUE TITLE
Internal Server Error 

### ISSUE THREAD

**Issue by linonetwo** [thundergolfer/automated-github-organization-invites#14]
https://tiddlywiki-chinese-team-invite.herokuapp.com/

logs:

```


2021-09-03T05:28:25.834941+00:00 app[web.1]: 	/app/vendor/bundle/ruby/2.6.0/gems/sinatra-1.4.8/lib/sinatra/base.rb:1787:in `synchronize'

2021-09-03T05:28:25.834941+00:00 app[web.1]: 	/app/vendor/bundle/ruby/2.6.0/gems/sinatra-1.4.8/lib/sinatra/base.rb:1487:in `call'

2021-09-03T05:28:25.834942+00:00 app[web.1]: 	/app/vendor/bundle/ruby/2.6.0/gems/rack-1.6.12/lib/rack/handler/webrick.rb:88:in `service'

2021-09-03T05:28:25.834942+00:00 app[web.1]: 	/app/vendor/ruby-2.6.6/lib/ruby/2.6.0/webrick/httpserver.rb:140:in `service'

2021-09-03T05:28:25.834942+00:00 app[web.1]: 	/app/vendor/ruby-2.6.6/lib/ruby/2.6.0/webrick/httpserver.rb:96:in `run'

2021-09-03T05:28:25.834947+00:00 app[web.1]: 	/app/vendor/ruby-2.6.6/lib/ruby/2.6.0/webrick/server.rb:307:in `block in start_thread'

2021-09-03T05:28:25.835231+00:00 app[web.1]: 222.72.40.10 - - [03/Sep/2021:05:28:25 +0000] "POST /add HTTP/1.1" 500 30 0.1003

2021-09-03T05:28:25.835462+00:00 app[web.1]: 10.1.32.186 - - [03/Sep/2021:05:28:25 UTC] "POST /add HTTP/1.1" 500 30

2021-09-03T05:28:25.835483+00:00 app[web.1]: https://automated-github-team-invite.herokuapp.com/ -> /add

2021-09-03T05:28:25.838832+00:00 heroku[router]: at=info method=POST path="/add" host=automated-github-team-invite.herokuapp.com request_id=6ac847ea-58c3-446e-bc65-c5dbee13434d fwd="222.72.40.10" dyno=web.1 connect=0ms service=102ms status=500 bytes=326 protocol=https


```

**Comment by pglevy** [comment]
I'm getting the same error here: https://join-gfn.herokuapp.com/

**Comment by linonetwo** [comment]
Use forks, forks works good.

**Comment by pglevy** [comment]
I was starting to go through some of them. Any particular one you recommend that works?

**Comment by thehassantahir** [comment]
> I'm getting the same error here: https://join-gfn.herokuapp.com/

same issue on both deployments

**Comment by michaelachrisco** [comment]
I tried the same thing and got the following error in heroku: 
```
bundle exec ruby web_app.rb/app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday/options.rb:160:in `new': tried to create Proc object without a block (ArgumentError)
        from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday/options.rb:160:in `memoized'
 $ ORGANfrom /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday/options.rb:241:in `<class:ProxyOptions>'
        from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday/options.rb:223:in `<module:Faraday>'
        from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday/options.rb:1:in `<top (required)>'
        from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday.rb:77:in `require'
        from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday.rb:77:in `block in require_libs'
        from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday.rb:76:in `each'
        from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday.rb:76:in `require_libs'
        from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday.rb:237:in `<module:Faraday>'
        from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday.rb:16:in `<top (required)>'
        from /app/vendor/bundle/ruby/3.0.0/gems/sawyer-0.8.1/lib/sawyer/agent.rb:1:in `require'
        from /app/vendor/bundle/ruby/3.0.0/gems/sawyer-0.8.1/lib/sawyer/agent.rb:1:in `<top (required)>'
        from /app/vendor/bundle/ruby/3.0.0/gems/sawyer-0.8.1/lib/sawyer.rb:17:in `require'
        from /app/vendor/bundle/ruby/3.0.0/gems/sawyer-0.8.1/lib/sawyer.rb:17:in `block in <top (required)>'
        from /app/vendor/bundle/ruby/3.0.0/gems/sawyer-0.8.1/lib/sawyer.rb:17:in `each'
        from /app/vendor/bundle/ruby/3.0.0/gems/sawyer-0.8.1/lib/sawyer.rb:17:in `<top (required)>'
        from /app/vendor/bundle/ruby/3.0.0/gems/octokit-4.6.2/lib/octokit/connection.rb:1:in `require'
        from /app/vendor/bundle/ruby/3.0.0/gems/octokit-4.6.2/lib/octokit/connection.rb:1:in `<top (required)>'
        from /app/vendor/bundle/ruby/3.0.0/gems/octokit-4.6.2/lib/octokit/client.rb:1:in `require'
        from /app/vendor/bundle/ruby/3.0.0/gems/octokit-4.6.2/lib/octokit/client.rb:1:in `<top (required)>'
        from /app/vendor/bundle/ruby/3.0.0/gems/octokit-4.6.2/lib/octokit.rb:1:in `require'
        from /app/vendor/bundle/ruby/3.0.0/gems/octokit-4.6.2/lib/octokit.rb:1:in `<top (required)>'
        from web_app.rb:5:in `require'
        from web_app.rb:5:in `<main>'
        ```

**Comment by thehassantahir** [comment]
> I tried the same thing and got the following error in heroku:
> 
> ```
> bundle exec ruby web_app.rb/app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday/options.rb:160:in `new': tried to create Proc object without a block (ArgumentError)
>         from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday/options.rb:160:in `memoized'
>  $ ORGANfrom /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday/options.rb:241:in `<class:ProxyOptions>'
>         from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday/options.rb:223:in `<module:Faraday>'
>         from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday/options.rb:1:in `<top (required)>'
>         from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday.rb:77:in `require'
>         from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday.rb:77:in `block in require_libs'
>         from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday.rb:76:in `each'
>         from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday.rb:76:in `require_libs'
>         from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday.rb:237:in `<module:Faraday>'
>         from /app/vendor/bundle/ruby/3.0.0/gems/faraday-0.11.0/lib/faraday.rb:16:in `<top (required)>'
>         from /app/vendor/bundle/ruby/3.0.0/gems/sawyer-0.8.1/lib/sawyer/agent.rb:1:in `require'
>         from /app/vendor/bundle/ruby/3.0.0/gems/sawyer-0.8.1/lib/sawyer/agent.rb:1:in `<top (required)>'
>         from /app/vendor/bundle/ruby/3.0.0/gems/sawyer-0.8.1/lib/sawyer.rb:17:in `require'
>         from /app/vendor/bundle/ruby/3.0.0/gems/sawyer-0.8.1/lib/sawyer.rb:17:in `block in <top (required)>'
>         from /app/vendor/bundle/ruby/3.0.0/gems/sawyer-0.8.1/lib/sawyer.rb:17:in `each'
>         from /app/vendor/bundle/ruby/3.0.0/gems/sawyer-0.8.1/lib/sawyer.rb:17:in `<top (required)>'
>         from /app/vendor/bundle/ruby/3.0.0/gems/octokit-4.6.2/lib/octokit/connection.rb:1:in `require'
>         from /app/vendor/bundle/ruby/3.0.0/gems/octokit-4.6.2/lib/octokit/connection.rb:1:in `<top (required)>'
>         from /app/vendor/bundle/ruby/3.0.0/gems/octokit-4.6.2/lib/octokit/client.rb:1:in `require'
>         from /app/vendor/bundle/ruby/3.0.0/gems/octokit-4.6.2/lib/octokit/client.rb:1:in `<top (required)>'
>         from /app/vendor/bundle/ruby/3.0.0/gems/octokit-4.6.2/lib/octokit.rb:1:in `require'
>         from /app/vendor/bundle/ruby/3.0.0/gems/octokit-4.6.2/lib/octokit.rb:1:in `<top (required)>'
>         from web_app.rb:5:in `require'
>         from web_app.rb:5:in `<main>'
>         ```
> ```

There Libs are out of update that's why we are facing issue.

**Comment by michaelachrisco** [comment]
Yep. @thehassantahir. 

I was able to get it working via a fork by @HID810 here: https://github.com/HIDE810/automated-github-organization-invites and fixed an issue where teams were nulling out here: https://github.com/michaelachrisco/automated-github-organization-invites/blob/master/web_app.rb#L91 

Should probably be backported at some point.

**Comment by thundergolfer** [comment]
Hey @michaelachrisco, yep I'll accept any PRs to fix these failures. Looks like code decay has significantly degraded this project's functionality.

**Comment by michaelachrisco** [comment]
Sounds good, we have a couple for your perusal and approval.

**Comment by thundergolfer** [comment]
I think these issues have been addressed in #16 and #17. I'll reopen if not.
